index.html

```diff
@@ -10,7 +10,48 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
     <script src="./node_modules/axios/dist/axios.min.js"></script>
+    <script src="./src/index.js"></script>
+    <!-- An element that displays the temperature
+A clickable element to increase temperature
+A clickable element to decrease temperature
+An element that displays a landscape -->
+
+<h1>Weather Report</h1>
+<p>for the lovely city of: <span id="headerCityName">Lousiville</span> </p>
+
+<section id="landscape">
+    <h2>Weather Garden</h2>
+    <p id="sky-container"></p>
+    <p id="landscape-container"></p>
+</section>
+
+<section id="temperature">
+    <h2>Temperature</h2>
+    <h3 id="temp-display">  </h3>
+    <button id="add-temp">Up</button>
+    <button id="sub-temp">Down</button>
+    <button id="real-time-temp">Current Temperature</button>
+</section>
+
+
+
+<section id = "sky">
+    <h2>Sky</h2>
+    <select id ="sky-menu">
+    <option>Cloudy</option>
+    <option>Sunny</option>
+    <option>Rainy</option>
+    <option>Snowy</option>
+</select>
```

Comment:
Going to be a little nit-picky: make sure your code is properly indented so it's easier to read and to debug! [The `option` HTML tag should also have a `value` field](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) to make these options accessible to screen readers: 
```
<select id="sky-menu">
    <option value="cloudy">Cloudy</option>
    <option value="sunny">Sunny</option>
    <option value="rainy">Rainy</option>
    <option value="snowy">Snowy</option>
</select>
```